velox/functions/prestosql/Probability.h

```diff
@@ -91,5 +91,24 @@ struct BinomialCDFFunction {
   }
 };
 
+template <typename T>
+struct InverseBetaCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(double& result, double a, double b, double p) {
+    constexpr double kInf = std::numeric_limits<double>::infinity();
```

Comment:
This can be a static variable.

Reply:
To clarify Aditi's comment use `static constexpr double kInf`

Reply:
Addressed in latest commit.